scripts/sbc/README.md

```diff
@@ -0,0 +1,3 @@
+# cp-demo Self Balancing Clusters
+
```

Comment:
@javabrett I understand that at some point docs may follow, but until then, perhaps the README can mention the scripts to run?  e.g. very brief mention of the key scripts, 

```
Run these scripts:

`./scripts/sbc/add-broker.sh`: add 3rd broker to force rebalancing
`./scripts/sbc/kill-broker.sh`: remove 3rd broker
```

WDYT?

Reply:
Excellent idea, done.